colcon_cmake/task/cmake/build.py

```diff
@@ -313,27 +311,24 @@ def _get_make_arguments(self):
     async def _install(self, args, env):
         self.progress('install')
 
-        generator = get_generator(args.build_base)
-        if 'Visual Studio' not in generator:
-            if CMAKE_EXECUTABLE is None:
-                raise RuntimeError("Could not find 'cmake' executable")
-            cmd = [
-                CMAKE_EXECUTABLE, '--build', args.build_base,
-                '--target', 'install']
+        if CMAKE_EXECUTABLE is None:
+            raise RuntimeError("Could not find 'cmake' executable")
+        cmd = [
+            CMAKE_EXECUTABLE, '--build', args.build_base,
+            '--target', 'install']
+        multi_configuration_generator = is_multi_configuration_generator(
+            args.build_base, args.cmake_args)
+        if multi_configuration_generator:
+            cmd += ['--config', self._get_configuration(args)]
+        else:
+            # Add make/ninja arguments for the number of job threads to use.
             job_args = self._get_make_arguments()
             if job_args:
+                # Add arguments for the number of job threads to use
+                # Arguably, _get_make_arguments() should be be configured to
```

Comment:
If you think this should be done please consider to contribute a pull request for it. This comment in the code is unlikely to be revisited hence I will remove it.